Source/ACE/Entity/Landblock.cs

```diff
@@ -417,6 +436,26 @@ public void UseTime()
                         HandleGameAction(action, p);
                 }
 
+                // despawn objects
+                Parallel.ForEach(despawnObjects, deo => 

```

Comment:
I don't think Parellel is worth the the thread overhead here.  You're not doing anything substantial in the loop.  Same with respawning creatures below.